src/module/rules/rule-element/item-alteration/schemas.ts

```diff
@@ -150,11 +150,14 @@ const ITEM_ALTERATION_VALIDATORS = {
     ),
     bulk: new ItemAlterationValidator({
         itemType: new fields.StringField({ required: true, choices: Array.from(PHYSICAL_ITEM_TYPES) }),
-        mode: new fields.StringField({ required: true, choices: ["override"] }),
-        value: new StrictNumberField<number, number, true, false, false>({
+        mode: new fields.StringField({
+            required: true,
+            choices: () => ["add", "downgrade", "multiply", "override", "remove", "subtract", "upgrade"],
+        }),
+        value: new StrictNumberField({
             required: true,
             nullable: false,
-            choices: [0, 0.1, ...Array.fromRange(100, 1)],
```

Comment:
We'll additional validation here as well to ensure the resulting value is valid.

Reply:
I already round the resulting value to 1/10ths, but I can add a check for it to be nonnegative, at least.

Reply:
0.1 (L) is also the only valid non-integer value since Light bulk isn't a countable unit of measurement.

Reply:
Technically correct, in practice, the system can actually handle values like "9L" or "3; 4L".

Reply:
:)